wsbroadcastserver/wsbroadcastserver.go

```diff
@@ -84,9 +84,12 @@ var DefaultTestBroadcasterConfig = BroadcasterConfig{
 }
 
 type WSBroadcastServer struct {
-	startMutex    *sync.Mutex
-	poller        netpoll.Poller
-	acceptDesc    *netpoll.Desc
+	startMutex *sync.Mutex
+	poller     netpoll.Poller
+
+	acceptDescMutex *sync.Mutex
```

Comment:
I generally prefer to not use a point for mutexes so then we can just use default initialization

Reply:
:+1: